src/mantle/testing/class-test-case.php

```diff
@@ -39,6 +39,8 @@
 
 /**
  * Root Test Case for Mantle sites.
+ *
+ * Not designed for external use. Use {@see Mantle\Testkit\Test_Case} instead.
```

Comment:
I can see a dev using this by mistake, especially if they have some IntelliSense in the editor where while importing the class, this note will not appear.

Would you consider adding this as part of custom namespace? Something like `Mantle\Testkit\Internal\Test_Case` or `Internal_Test_Case`...

Reply:
I can, I will rename it in a follow up